src/sdk/modules/validators/k1Validator.test.ts

```diff
@@ -70,57 +70,84 @@ describe("modules.k1Validator.write", async () => {
     expect(balanceAfter - balanceBefore).toBe(1n)
   })
 
-  test.skip("should install k1 validator with 1 owner", async () => {
+  test("should install k1 validator with 1 owner", async () => {
     const isInstalledBefore = await nexusClient.isModuleInstalled({
       module: {
         type: "validator",
         address: addresses.K1Validator,
-        context: "0x"
+        data: encodePacked(["address"], [account.address])
       }
     })
+    console.log(isInstalledBefore, "isInstalledBefore");
 
     if (!isInstalledBefore) {
       const hash = await nexusClient.installModule({
         module: {
           address: addresses.K1Validator,
           type: "validator",
-          context: "0x"
+          data: encodePacked(["address"], [account.address])
         }
       })
 
       const { success: installSuccess } =
         await nexusClient.waitForUserOperationReceipt({ hash })
       expect(installSuccess).toBe(true)
 
-      const hashUninstall = await nexusClient.uninstallModule({
+      const [installedValidators] = await nexusClient.getInstalledValidators({})
+
+      const prevModule = await nexusClient.getPreviousModule({
+        module: {
+          address: addresses.K1Validator,
+          type: "validator"
+        },
+        installedValidators
+      })
+
+      const deInitData = encodeAbiParameters(
+        [
+          { name: "prev", type: "address" },
+          { name: "disableModuleData", type: "bytes" }
+        ],
+        [prevModule, encodePacked(["address"], [account.address])]
+      )
+
+      const hashUninstall = nexusClient.uninstallModule({
         module: {
           address: addresses.K1Validator,
           type: "validator",
-          context: "0x"
+          data: deInitData
```

Comment:
here the 'getData' helper from the uninstallModule file could find the deInitData?